diff --git a/Source/App/CMakeLists.txt b/Source/App/CMakeLists.txt
index d305a55..b4d5ee3 100644
--- a/Source/App/CMakeLists.txt
+++ b/Source/App/CMakeLists.txt
@@ -51,9 +51,6 @@ target_link_libraries(SvtAv1EncApp
 if(${CMAKE_SYSTEM_NAME} STREQUAL "Android")
     target_link_libraries(SvtAv1EncApp ${PLATFORM_LIBS})
 elseif(UNIX)
-    target_link_libraries(SvtAv1EncApp
-        pthread
-        m)
 endif()
 
 install(TARGETS SvtAv1EncApp RUNTIME DESTINATION ${CMAKE_INSTALL_FULL_BINDIR})
